lib/matplotlib/colors.py

```diff
@@ -1410,7 +1440,7 @@ def autoscale_None(self, A):
             self.vmax = A.max()
 
     def scaled(self):
-        """Return whether vmin and vmax are set."""
+        """Return ``True`` if both vmin and vmax are set, return ``False`` otherwise."""
```

Comment:
I feel quite strongly that these methods that return a bool should be phrased using "Return whether" as much as possible, otherwise the second half (return False ...) is basically redundant.  @melissawm @story645 perhaps this can be an entry in the doc style guide?

Reply:
"whether both vmin and vmax are set"?

Reply:
sure

Reply:
I think with "both" this looks good.